### ISSUE TITLE
Nope

### ISSUE THREAD

**Issue by Kwenma1** [Leeon123/TCP-UDP-Flood#2]
It just ddosed urself and not the server lmao

**Comment by diegohtml** [comment]
?

Pada tanggal Rab, 29 Des 2021 17.46, Kwenma1 ***@***.***>
menulis:

**Comment by Kwenma1** [comment]
Yeah I wrote this a long time ago now Ik that it will harm my connection

**Comment by diegohtml** [comment]
I don't know, why that u tools works for DDOS, i expect that, and using u
client for DDOS server SA-MP, u know?


U can search yt GTA SAMP, And, u tools works that, sorry for abuse :)

**Comment by diegohtml** [comment]
Sorry for low respond, cause u email short
